tests/bgp/test_bgp_speaker.py

```diff
@@ -56,7 +54,11 @@ def common_setup_teardown(duthost, ptfhost, localhost):
     mg_facts = duthost.minigraph_facts(host=duthost.hostname)['ansible_facts']
     interface_facts = duthost.interface_facts()['ansible_facts']
 
-    res = duthost.shell("sonic-cfggen -m -d -y /etc/sonic/constants.yml -v \"constants.deployment_id_asn_map[DEVICE_METADATA['localhost']['deployment_id']]\"")
+    check = duthost.shell("test -e /etc/sonic/constants.yml || echo 'not present'")
```

Comment:
use stat module in ansible?

https://github.com/Azure/sonic-mgmt/pull/1808/files#diff-c1585dfa9e246a2432d256d6a667d5cdR84